tests/benchdnn/utils/numeric.hpp

```diff
@@ -36,5 +36,21 @@ bool is_integral_dt(dnnl_data_type_t dt);
 float maybe_saturate(dnnl_data_type_t dt, float value);
 float round_to_nearest_representable(dnnl_data_type_t dt, float value);
 bool is_subbyte_type(const dnnl_data_type_t &type);
-
+template <class T>
+T gcd(std::initializer_list<T> ilist) {
+    auto gcd = [](T a, T b) {
+        if (b == 0) return a;
+        while (a % b != 0) {
+            auto tmp = b;
+            b = a % b;
+            a = tmp;
+        }
+        return b;
+    };
+    if (ilist.size() < 1) return 0;
+    T ret = *ilist.begin();
```

Comment:
```suggestion
    T ret = 0;
```

Reply:
It will work with `0` as well, see [here](https://ideone.com/F10lNO) as `gcd(0, a) = a`.